apps/mull-ui/src/app/components/chat-bubble-list/chat-bubble-list.tsx

```diff
@@ -0,0 +1,30 @@
+import { Post } from 'apps/mull-ui/src/generated/graphql';
+import { avatarUrl } from 'apps/mull-ui/src/utilities';
+import React from 'react';
+import ChatBubble from '../../components/chat-bubble/chat-bubble';
+
+const formatDate = (date: Date) => {
+  return `${date.getHours()}:${String(date.getMinutes()).padStart(
+    2,
+    '0'
+  )} ${date.toLocaleDateString()}`;
+};
+
```

Comment:
As @bit172 points out below, this component should accept a list of `posts` directly. You'll need to add an interface so that you have the correct types

```suggestion
export interface ChatBubbleListProps {
  posts: Post[];
}
```